src/Symfony/Component/HttpClient/Response/AsyncResponse.php

```diff
@@ -272,6 +272,14 @@ public static function stream(iterable $responses, float $timeout = null, string
                     continue;
                 }
 
+                if (null !== $chunk->getError()) {
+                    // no-op
+                } elseif ($chunk->isFirst()) {
+                    $r->yieldedState = 1;
```

Comment:
should we use private constants for those states to make it easier to understand what they represent ?

Reply:
ok, updated